[0m[[0minfo[0m] [0mRunning Main [0m
[0m[[0mdebug[0m] [0mWaiting for threads to exit or System.exit to be called.[0m
[0m[[0mdebug[0m] [0mWaiting for thread run-main-0 to terminate.[0m
[0m[[0mdebug[0m] [0m  Classpath:[0m
[0m[[0mdebug[0m] [0m	C:\Users\aman\workspace\local\target\scala-2.11\classes[0m
[0m[[0mdebug[0m] [0m	C:\Users\aman\.ivy2\cache\org.scala-lang\scala-library\jars\scala-library-2.11.7.jar[0m
[0m[[0mdebug[0m] [0m	C:\Users\aman\.ivy2\cache\com.typesafe.akka\akka-actor_2.11\jars\akka-actor_2.11-2.4.0-RC2.jar[0m
[0m[[0mdebug[0m] [0m	C:\Users\aman\.ivy2\cache\com.typesafe\config\bundles\config-1.3.0.jar[0m
[0m[[0mdebug[0m] [0m	C:\Users\aman\.ivy2\cache\com.typesafe.akka\akka-remote_2.11\jars\akka-remote_2.11-2.4.0-RC2.jar[0m
[0m[[0mdebug[0m] [0m	C:\Users\aman\.ivy2\cache\com.typesafe.akka\akka-protobuf_2.11\jars\akka-protobuf_2.11-2.4.0-RC2.jar[0m
[0m[[0mdebug[0m] [0m	C:\Users\aman\.ivy2\cache\org.openjdk.jmh\jmh-core\jars\jmh-core-1.10.3.jar[0m
[0m[[0mdebug[0m] [0m	C:\Users\aman\.ivy2\cache\net.sf.jopt-simple\jopt-simple\jars\jopt-simple-4.6.jar[0m
[0m[[0mdebug[0m] [0m	C:\Users\aman\.ivy2\cache\org.apache.commons\commons-math3\jars\commons-math3-3.2.jar[0m
[0m[[0mdebug[0m] [0m	C:\Users\aman\.ivy2\cache\org.openjdk.jmh\jmh-generator-bytecode\jars\jmh-generator-bytecode-1.10.3.jar[0m
[0m[[0mdebug[0m] [0m	C:\Users\aman\.ivy2\cache\org.openjdk.jmh\jmh-generator-reflection\jars\jmh-generator-reflection-1.10.3.jar[0m
[0m[[0mdebug[0m] [0m	C:\Users\aman\.ivy2\cache\org.openjdk.jmh\jmh-generator-asm\jars\jmh-generator-asm-1.10.3.jar[0m
[0m[[0mdebug[0m] [0m	C:\Users\aman\.ivy2\cache\org.ow2.asm\asm\jars\asm-5.0.3.jar[0m
[0m[[0mdebug[0m] [0m	C:\Users\aman\.ivy2\cache\io.netty\netty\bundles\netty-3.10.3.Final.jar[0m
[0m[[0mdebug[0m] [0m	C:\Users\aman\.ivy2\cache\org.uncommons.maths\uncommons-maths\jars\uncommons-maths-1.2.2a.jar[0m
[0m[[31merror[0m] [0m(run-main-0) java.lang.ArrayIndexOutOfBoundsException: 0[0m
java.lang.ArrayIndexOutOfBoundsException: 0
	at Main$.delayedEndpoint$Main$1(local.scala:451)
	at Main$delayedInit$body.apply(local.scala:445)
	at scala.Function0$class.apply$mcV$sp(Function0.scala:34)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
	at scala.App$$anonfun$main$1.apply(App.scala:76)
	at scala.App$$anonfun$main$1.apply(App.scala:76)
	at scala.collection.immutable.List.foreach(List.scala:381)
	at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:35)
	at scala.App$class.main(App.scala:76)
	at Main$.main(local.scala:445)
	at Main.main(local.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at sbt.Run.invokeMain(Run.scala:67)
	at sbt.Run.run0(Run.scala:61)
	at sbt.Run.sbt$Run$$execute$1(Run.scala:51)
	at sbt.Run$$anonfun$run$1.apply$mcV$sp(Run.scala:55)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:55)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:55)
	at sbt.Logger$$anon$4.apply(Logger.scala:85)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Unknown Source)
[0m[[0mdebug[0m] [0m	Thread run-main-0 exited.[0m
[0m[[0mdebug[0m] [0mWaiting for thread LocalSystem-akka.actor.default-dispatcher-5 to terminate.[0m
[0m[[0mdebug[0m] [0m	Thread LocalSystem-akka.actor.default-dispatcher-5 exited.[0m
[0m[[0mdebug[0m] [0mWaiting for thread HelloSystem-scheduler-1 to terminate.[0m
